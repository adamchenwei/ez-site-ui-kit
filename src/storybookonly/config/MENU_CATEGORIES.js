// Atomic Design Component Structure
export const COMPONENT = 'Component';
export const ATOM = 'Component';
export const MOLECULE = 'Molecule';
export const ORGANISM = 'Organism';
export const TEMPLATE = 'Template';
export const PAGE = 'Page';
// Framework Base Options
export const BOOTSTRAP = 'Bootstrap';
export const MATERIALUI = 'Material-ui';
export const PURE = 'Pure';

// Containers for Redux or Global State
export const CONTAINER = 'Container';

export default {
  COMPONENT,
  ATOM,
  MOLECULE,
  ORGANISM,
  TEMPLATE,
  PAGE,
  BOOTSTRAP,
  MATERIALUI,
  PURE,
  CONTAINER,
};
