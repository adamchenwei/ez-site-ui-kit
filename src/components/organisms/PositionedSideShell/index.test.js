import React from 'react';
import renderer from 'react-test-renderer';
import PositionedSlideShell from './index';
import { COMPONENT_LABEL, ES6 } from '../../../storybookonly/config/MENU_CATEGORIES';

describe(`${COMPONENT_LABEL}/${ES6}/PositionedSlideShell/Unit Test`, () => {
  test('renders as expected', () => {
    const tree = renderer.create(<PositionedSlideShell />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
