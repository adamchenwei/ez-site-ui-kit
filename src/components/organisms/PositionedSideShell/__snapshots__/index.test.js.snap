// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/ES6/PositionedSlideShell/Unit Test renders as expected 1`] = `
<section
  className="sc-htoDjs dnOwue"
>
  <section
    className="sc-bdVaJa kZpTwA"
    style={null}
  >
    <section
      className="sc-htpNat jILPxC"
    >
      <span
        className="sc-ifAKCX fYEscZ"
      >
        
      </span>
      <span
        className="sc-EHOje epUoHW"
      >
        
      </span>
      <span
        className="sc-bZQynM iVzJoL"
      >
        
      </span>
    </section>
    <section
      className="sc-gzVnrw ecJclC"
    >
      <img
        alt="slide-background"
        className="sc-bxivhb haYJEf"
        src={null}
      />
    </section>
  </section>
</section>
`;
